locales/en-US/strings.properties

```diff
@@ -4,18 +4,20 @@ default_label_loading=Loading…
 header_top_sites=Top Sites
 header_highlights=Highlights
 
-# LOCALIZATION NOTE(type_label_*): These labels are shown with a page to give
-# context on how the page is related to the user
-# e.g. The type indicates that the page is bookmarked, or is currently open on
-# another device
+# LOCALIZATION NOTE (type_label_*): These labels are associated to pages to give
+# context on how the element is related to the user, e.g. type indicates that
+# the page is bookmarked, or is currently open on another device
 type_label_visited=Visited
 type_label_bookmarked=Bookmarked
 type_label_synced=Synced from another device
+# LOCALIZATION NOTE(type_label_open): Open is an adjective, as in "page is open"
```

Comment:
Rationale for adding a comment before the problematic string is that tools will only display the general comment with the following string, not the entire group.